pkg/agent/proxy/proxier.go

```diff
@@ -342,6 +376,21 @@ func (p *proxier) installServices() {
 				klog.Errorf("Error when installing Endpoints groups: %v", err)
 				continue
 			}
+			// Install another group for local type NodePort service.
+			if p.nodePortEnabled && svcInfo.NodePort() > 0 && svcInfo.NodeLocalExternal() {
+				nGroupID, _ := p.groupCounter.Get(svcPortName, endpointLocalLabel)
```

Comment:
Question - how about add another groupCounter for local NodePort Services?

Reply:
I guess you mean that adding another groupCounter struct to the structure of proxier?

Reply:
Yes, compared including endpointLocalLabel in the key.

Reply:
If so, these two groupCounter struct should also share the same map to store the existing key:value of `Service name:group number` as the group number is used to created a lb group. The lb group will be used as action in the flows of pipeline table 41. Group number should be duplicated.

Reply:
I see. Unless we allocate a separate range for local NodePort. Maybe not worthwhile.

Reply:
Could we pass a boolean to groupCounter.Get() and add a suffix like "/L" to the key? Feel no need to pass the label string to generate keys, as we might not have other cases to include an label.

Reply:
I see. This could be better.